mflowgen/test/test_module.sh

```diff
@@ -407,16 +456,17 @@ for step in ${build_sequence[@]}; do
         exit 13
     fi
 
-    # Optionally update (gold) cache after each step
-    if [ "$update_cache" ]; then
-        gold="$update_cache"; # E.g. gold="/sim/buildkite-agent/gold.13"
-        echo "+++ SAVE RESULTS so far to cache '$gold'"
-        test -d $gold || mkdir $gold
-        set -x
-        cp -rpf $build/full_chip $gold
-        set +x
-        ls -l $gold/full_chip || PASS
-    fi
+# Don't need this no more maybe
+#     # Optionally update (gold) cache after each step
+#     if [ "$update_cache" ]; then
+#         gold="$update_cache"; # E.g. gold="/sim/buildkite-agent/gold.13"
+#         echo "+++ SAVE RESULTS so far to cache '$gold'"
+#         test -d $gold || mkdir $gold
+#         set -x
+#         cp -rpf $build/full_chip $gold
+#         set +x
+#         ls -l $gold/full_chip || PASS
+#     fi
```

Comment:
I suppose this isn't needed because the cache directory is just the build directory. If so, we should just delete this rather than commenting it out.

Reply:
You are right. I will try and remember to delete it on the next iteration (should be soon).

Reply:
Why not just delete it for this PR?